ako-gateway-api/k8s/gateway_controller.go

```diff
@@ -188,7 +188,7 @@ func (c *GatewayController) SetupEventHandlers(k8sinfo k8s.K8sinformers) {
 				if c.DisableSync {
 					return
 				}
-				ep := obj.(*corev1.Endpoints)
+				ep := obj.(*corev1.Endpoints) //nolint:staticcheck
```

Comment:
why is static check disabled for this line

Reply:
References: https://kubernetes.io/blog/2025/04/24/endpoints-deprecation/